src/Options.ts

```diff
@@ -4,6 +4,11 @@ import { hasMagic as hasGlob, sync as globSync } from 'glob';
 import { basename, extname, resolve } from 'path';
 import { sync as resolveSync } from 'resolve';
 import { PathPredicate } from './iterateSources';
+import PluginLoader from './PluginLoader';
```

Comment:
Here’s where the big changes start. Rather than making loading the responsibility of `Plugin`, I created a `PluginLoader` class that takes a series of resolvers that can resolve things that are given on the command line (i.e. `my-plugin` from `--plugin my-plugin`) and turn them into file paths to be loaded. Once the file paths are resolved, `PluginLoader` does a `require` and extracts the babel plugin.